### ISSUE TITLE
Commit #07feec8df3927f3c150dd0cf7db9a54927bd2569 causes build errors on Intel

### ISSUE THREAD

**Issue by mharshe** [google/XNNPACK#1390]
The latest commit causes the following build errors on intel machines. (Pasting only the first few lines here, can provide more if required):
```
[141/425] Building C object CMakeFiles/XNNPACK.dir/src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c.o
FAILED: CMakeFiles/XNNPACK.dir/src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c.o 
/usr/bin/cc -DCPUINFO_SUPPORTED_PLATFORM=1 -DFXDIV_USE_INLINE_ASSEMBLY=0 -DPTHREADPOOL_NO_DEPRECATED_API=1 -DXNN_ENABLE_ASSEMBLY=1 -DXNN_ENABLE_MEMOPT=1 -DXNN_ENABLE_SPARSE=1 -DXNN_LOG_LEVEL=0 -I../../include -I../../src -Iclog-source/deps/clog/include -Icpuinfo-source/include -Ipthreadpool-source/include -IFXdiv-source/include -IFP16-source/include -O3 -DNDEBUG -fPIC   -Wno-psabi -pthread -std=gnu99  -msse4.1  -O2 -MD -MT CMakeFiles/XNNPACK.dir/src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c.o -MF CMakeFiles/XNNPACK.dir/src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c.o.d -o CMakeFiles/XNNPACK.dir/src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c.o   -c ../../src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c
../../src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c: In function ‘xnn_qs8_dwconv_minmax_ukernel_up8x9__sse41_mul32’:
../../src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c:87:50: warning: implicit declaration of function ‘_mm_loadu_si32’; did you mean ‘_mm_loadu_si128’? [-Wimplicit-function-declaration]
       const __m128i vi0x0123 = _mm_cvtepi8_epi32(_mm_loadu_si32(i0));
                                                  ^~~~~~~~~~~~~~
                                                  _mm_loadu_si128
../../src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c:87:50: error: incompatible type for argument 1 of ‘_mm_cvtepi8_epi32’
In file included from /usr/lib/gcc/x86_64-linux-gnu/7/include/immintrin.h:37:0,
                 from ../../src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c:12:
/usr/lib/gcc/x86_64-linux-gnu/7/include/smmintrin.h:482:1: note: expected ‘__m128i {aka __vector(2) long long int}’ but argument is of type ‘int’
 _mm_cvtepi8_epi32 (__m128i __X)
 ^~~~~~~~~~~~~~~~~
../../src/qs8-dwconv/gen/up8x9-minmax-sse41-mul32.c:88:50: error: incompatible type for argument 1 of ‘_mm_cvtepi8_epi32’
```
I have verified that the previous commit works well.

**Comment by huningxin** [comment]
same error here

**Comment by Maratyszcza** [comment]
Expected to be fixed in #1391
